lte/gateway/python/integ_tests/Makefile

```diff
@@ -36,21 +36,19 @@ define execute_test
 	# Testcase $(1) execution completed
 endef
 
+.PHONY: selected_tests
+selected_tests: $(PYTHON_BUILD)/setupinteg_env $(BIN)/pytest
+	. $(PYTHON_BUILD)/bin/activate
+	$(foreach test,$(TESTS),$(call execute_test,$(test));)
+
 .PHONY: precommit
 precommit: $(PYTHON_BUILD)/setupinteg_env $(BIN)/pytest
 	. $(PYTHON_BUILD)/bin/activate
-ifdef TESTS
-	$(call execute_test,$(TESTS))
-else
 	$(foreach test,$(PRECOMMIT_TESTS),$(call execute_test,$(test));)
-endif
 
 .PHONY: integ_test
 integ_test: $(PYTHON_BUILD)/setupinteg_env $(BIN)/pytest
 	. $(PYTHON_BUILD)/bin/activate
-ifdef TESTS
-	$(call execute_test,$(TESTS))
-else
 	echo "pass" > $(MAGMA_ROOT)/test_status.txt
```

Comment:
This should not just be removed without deprecation warning or something similar since its usage is actively promoted in the AGW S1AP integration test [docs](https://docs.magmacore.org/docs/lte/s1ap_tests#run-tests) and used on a daily basis to execute single integration tests.

Reply:
As discussed, the second commit on this PR updates the documentation.